### PR TITLE
[Chore] Bump morley

### LINKED ISSUE #182: Remove `StablecoinImpl m` capability
# Clarification and motivation

Thanks to the recent changes to `cleveland`, stablecoin doesn't actually need a `StablecoinImpl m` capability anymore.

All stablecoin needs to do is grab access to a `MorleyClientEnv` and be able to run in `IO`.
We can do that with `runIO` and `whenNetworkEnabled`, so stablecoin does not need any extra capabilities. E.g.:

```hs
test_demo :: TestTree
test_demo =
  whenNetworkEnabled \withEnv -> do
    testScenario "demo" $ scenario do
      runIO $ withEnv \env -> callStablecoinClient env ["deploy"]
```

This should make things much simpler.

# Acceptance criteria

The functions in `Stablecoin.Client.Cleveland.Caps`:
* Don't use `MonadStablecoin` or `StablecoinImpl m` anymore
* They run on `MonadCleveland`

### PR DESCRIPTION
**Pull request by lierdakil**
Draft because uses indigo &c from branches

## Description

Problem: new morley &c are out

Solution: update

Problem: no longer required, we can call stablecoin client from network
scenarios directly.

Solution: remove the capability, rework helper functions to run in
MonadNetwork.

## Related issue(s)

Resolves #182 

Part of https://issues.serokell.io/issue/TM-674

#### Related changes (conditional)

- Tests
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - [ ] I checked whether I should update the docs and did so if necessary:
    - [README](https://github.com/tqtezos/stablecoin/tree/master/README.md)
    - Haddock
  - [ ] I updated [changelog](https://github.com/tqtezos/stablecoin/tree/master/ChangeLog.md) unless I am sure my changes are
        not essential.

#### Stylistic guide (mandatory)

- [ ] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Where-and-how-to-commit-your-work-58f8973a4b3142c8abbd2e6fd5b3a08e).
- [ ] My code complies with the [style guide](https://github.com/tqtezos/stablecoin/tree/master/docs/code-style.md).

#### ✓ Release Checklist

- [ ] I updated the [Gas / Transaction costs](https://github.com/tqtezos/stablecoin/tree/master/README.md#gas--transaction-costs) table in the README.
- [ ] I updated the version number in `package.yaml`.
- [ ] I updated the [changelog](https://github.com/tqtezos/stablecoin/tree/master/ChangeLog.md) and moved everything
      under the "Unreleased" section to a new section for this release version.
- [ ] (After merging) I created a new entry in the [releases](https://github.com/tqtezos/stablecoin/releases) page,
      with a summary of all user-facing changes.
    * I made sure a tag was created using the format `vX.Y.Z`
    * I attached the following artifacts from [buildkite](https://buildkite.com/serokell/stablecoin)
      * `stablecoin-client`
      * `stablecoin.fa1.2.tz`
      * `stablecoin.tz`

### CODE REVIEW

File: haskell/test/Lorentz/Contracts/Nettest/Nettest.hs [hunk 1]
```diff
@@ -1,6 +1,9 @@
 -- SPDX-FileCopyrightText: 2021 Oxhead Alpha
 -- SPDX-License-Identifier: MIT
 
+-- for Ord ImplicitAddressWithAlias
```
Comment (pasqu4le) [inline haskell/test/Lorentz/Contracts/Nettest/Nettest.hs:4]: Perhaps it's worth doing this change in Morley?

**Review by pasqu4le — APPROVED** [review]
LGTM, pending the commit updates in `stack.yaml`

File: haskell/src/Stablecoin/Client/Cleveland/Caps.hs [hunk 2]
```diff
@@ -37,200 +34,291 @@ module Stablecoin.Client.Cleveland.Caps
...
-deploy s st = implActionToCaps \cap -> siDeploy cap s st
+data StablecoinTestError where
+  STEDiff :: forall a. (Show a, Buildable a) => a -> a -> StablecoinTestError
+
+deriving stock instance Show StablecoinTestError
+
+instance Buildable StablecoinTestError where
+  build (STEDiff actual expected) =
+    "--- Diff ---\n" <>
+    "Expected: " <> pretty expected <> "\n" <>
```
[the remark this answers is not in the document]
Comment (lierdakil) [reply haskell/src/Stablecoin/Client/Cleveland/Caps.hs:62]: FWIW, this was moved from `StablecoinImpl.hs` verbatim, so this isn't new.

File: haskell/test/Lorentz/Contracts/Nettest/Nettest.hs (hunk 1 above)
Comment (pasqu4le) [reply haskell/test/Lorentz/Contracts/Nettest/Nettest.hs:4]: I also don't feel strongly either way, but I've opened [morley/#936](https://gitlab.com/morley-framework/morley/-/issues/936) to do this, as it doesn't hurt.

If we change our mind that issue can always be closed too

**Pull request merged**